### PR TITLE
Adding more info to Contributor Guide

### PR DESCRIPTION
**Pull request by sdwheeler**
Described how to link to content across docs.microsoft.com. Added note about OPS alert extensions (NOTE, WARNING, IMPORTANT, TIP).

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[contributing/STYLE.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-opsext/contributing/STYLE.md) | :white_check_mark:Succeeded | |
[contributing/WRITING.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-opsext/contributing/WRITING.md) | :white_check_mark:Succeeded | |
[contributing/images/alert-boxes.png](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-opsext/contributing/images/alert-boxes.png) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C9%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201801091832520081-2023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=UWXJM1uE6SxM6Ia4IJzQpsuripJcQXU8IBQHvVUGBHY%3D&st=2018-01-09T18%3A55%3A09Z&se=2018-02-09T19%3A00%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: contributing/STYLE.md [hunk 1]
```diff
@@ -111,19 +111,94 @@ to get this output:
...
+
+```Markdown
+[Overview of Azure PowerShell](/powershell/azure/overview)
+```
+
+Here is the live link:
+
+[Overview of Azure PowerShell](/powershell/azure/overview)
+
+Note that the live link does not within the GitHub view of this content.
```
Comment (zjalexander) [inline contributing/STYLE.md]: "live link does not within"

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[contributing/STYLE.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-opsext/contributing/STYLE.md) | :white_check_mark:Succeeded | |
[contributing/WRITING.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-opsext/contributing/WRITING.md) | :white_check_mark:Succeeded | |
[contributing/images/alert-boxes.png](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-opsext/contributing/images/alert-boxes.png) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C9%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201801092311473053-2023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=C8nr9w%2FDNYItv7z%2BfOn7vj8u76ltHQSTVfXpUI%2BTNxA%3D&st=2018-01-09T23%3A33%3A59Z&se=2018-02-09T23%3A38%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[contributing/STYLE.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-opsext/contributing/STYLE.md) | :white_check_mark:Succeeded | |
[contributing/WRITING.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-opsext/contributing/WRITING.md) | :white_check_mark:Succeeded | |
[contributing/images/alert-boxes.png](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-opsext/contributing/images/alert-boxes.png) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C10%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201801092311571342-2023%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=rL57aoZHk1RIXU9E7qijMM3VRWQ9%2BuKmuvTFeRmHAoM%3D&st=2018-01-10T00%3A01%3A06Z&se=2018-02-10T00%3A06%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**